### PR TITLE
Send message on online users in Demisto

### PR DESCRIPTION
**Pull request by gshriki**
a script that sends message to online Demisto users

### CODE REVIEW

File: Scripts/script-SendMessageToOnlineUsers.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  }
+
+  return entries;
+type: javascript
+tags: []
+comment: Send message to Demisto online users over Email, Slack or both.
+enabled: true
+args:
+- name: medium
+  auto: PREDEFINED
```
Comment (aviadl) [inline Scripts/script-SendMessageToOnlineUsers.yml]: Add a default value ?

File: Scripts/script-SendMessageToOnlineUsers.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          instanceKeys.forEach(function (key) {
+              if (allModules[key].state === 'active') {
+                  ret = true;
+              }
+          });
+      }
+      return ret;
+  };
+  var slackEnabled = commandAvailable('slack-send');
+  var emailEnabled = commandAvailable('send-mail');
```
Comment (aviadl) [inline Scripts/script-SendMessageToOnlineUsers.yml]: Do you want to return here an error in case the medium selected is not enabled?

Comment (gshriki) [reply Scripts/script-SendMessageToOnlineUsers.yml]: If it is all - than I'll only send to the enabled ones. If it is specific - there will be an error anyway.
I will add an error if none of the mediums are available.

File: Scripts/script-SendMessageToOnlineUsers.yml (hunk 1 above)
Comment (gshriki) [reply Scripts/script-SendMessageToOnlineUsers.yml]: done

File: Scripts/script-SendMessageToOnlineUsers.yml (hunk 2 above)
Comment (aviadl) [reply Scripts/script-SendMessageToOnlineUsers.yml]: But you will have multiple errors of the same type, which will only create confusion.
So why not create one error here (**not** for case of **all**)

Comment (gshriki) [reply Scripts/script-SendMessageToOnlineUsers.yml]: got you. agree. added.

**Pull request merged**